### ISSUE TITLE
atm-meta

### ISSUE THREAD

**Issue by Olivash** [atmos-python/atmos#12]
Hi so I was writing a code and I used atm.meta(data) and I am getting the error "AttributeError: module 'atmos' has no attribute 'meta'".  Can anyone please help?

**Comment by mcgibbon** [comment]
Yes, what are you trying to do? `meta` is not something that exists in `atmos` (and I can't tell what it's supposed to be).

**Comment by Olivash** [comment]
I'm trying to do a composite analysis for 10m wind.

**Comment by mcgibbon** [comment]
If possible, can you please frame the issue as a [Minimal, Complete, Reproducible example](https://stackoverflow.com/help/minimal-reproducible-example)? It is still unclear to me what you are trying to do and what the purpose of `atm.meta(data)` is supposed to be (what's the expected output and why?).
